src/index.js

```diff
@@ -18,7 +18,7 @@ const sendMessage = async function (opts) {
   const files = opts.files || []
   const filtered = opts.filter || true
 
-  if (filtered && checks.isDirty(content)) return
+  if (filtered && checks.isDirty(content) && !checks.isAdmin(userID)) return
```

Comment:
I'm confused what you're trying to do here.
This (in theory) would just make it so mods can always get a response to their command?
However, the userID variable also isn't in this scope, so this would just throw an exception anyway.

The actual issue I think you're trying to fix is that balances aren't showing up when there's a bad word in them.
The logic is messed up in this function actually - we pass a `filter` boolean on L158, however L19 isn't actually respecting it. Take a look there. ;)